code/subscription.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kreis4 Flower Shop - Order Flowers Online</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
+    <menu>
+        <a href="./index.html">Home</a>
+        <a href="./our-flowers.html">Flowers</a>
+        <a href="./subscription.html">Subscription</a>
+        <a href="./faqs.html">FAQs</a>
+        <a href="./about.html">About Us</a>
+    </menu>
+
+<p class="main-block" style="margin:30px; font-size:38px;"><em>under construction</em></p>
+
+<br>
+<div style="width: 500px;" class="highlighter-block">
+  <h2>Subscribe here:</h2>
+<p>
+  Email Address: <input type="text"><br>
+  Your Subscription:
+  <div style="padding-left:20px;margin:15px;">
+  <input type="checkbox"> Weekly Subscription<br>
+  <input type="checkbox"> Bi-Weekly Subscription<br>
+  <input type="checkbox"> Monthly subscription<br>
+</div>
+  Start Date: <input type="date"><br>
+  </p>
+
+  <button>Submit</button>
+</div>
```

Comment:
Yey, a form, you're ahead 🤩 There's an HTML element called `form` that you could use instead of div, and remember the indentation (and to keep the styling in the CSS-file). Also, there's an element called label that you could use to label your different form elements. Take a look [here](https://www.w3schools.com/tags/tag_label.asp)!